src/Core/Vipr.Core/CodeModel/OdcmObject.cs

```diff
@@ -11,5 +13,30 @@ public OdcmObject(string name)
         {
             Name = name;
         }
+
```

Comment:
Actually, these are canonical names. While they may resemble names from an arbitrary language, such as C#, they are not meant to be used as such. For example, an OdcmMethod's canonical name is given as the concatenation of its parameter types, return value, and name, e.g., "violet.Strike(violet.Quake,Edm.String,Edm.Int32):Edm.DateTimeOffset".